src/Misc/layoutbin/update.sh.template

```diff
@@ -180,6 +180,9 @@ fi
 
 date "+[%F %T-%4N] Update succeed" >> "$logfile"
 
+touch update.finished
```

Comment:
Should we make sure to best effort cleanup this file if it already exists?

Reply:
`run-helper.sh` will always wait for `Runner.Listener` to finish before attempting a restart.

We should probably clean up update.finished in `Runner.Listener` right before it starts a process with the generated `_update.sh`. This way `update.finished` is cleaned up before `_update.sh` starts, so no way for the shell script to generate the update file, and before `Runner.Listener` stops so `run-helper.sh` for sure doesn't attempt a restart.